stuf_zds_payments/tests/test_plugin.py

```diff
@@ -0,0 +1,201 @@
+from decimal import Decimal
+from pathlib import Path
+
+from django.test import TestCase
+
+from openforms.payments.constants import PaymentStatus
+from openforms.payments.tests.factories import SubmissionPaymentFactory
+from openforms.registrations.constants import RegistrationAttribute
+from openforms.submissions.tests.factories import SubmissionFactory
+from openforms.utils.tests.vcr import OFVCRMixin
+from stuf.stuf_zds.models import StufZDSConfig
+from stuf.tests.factories import StufServiceFactory
+
+from stuf_zds_payments.client import ZaakOptions
+from stuf_zds_payments.plugin import (
+    StufZDSPaymentsRegistration,
+    default_payment_status_update_mapping,
+)
+
+TESTS_DIR = Path(__file__).parent.resolve()
+
+
+class StufZDSPaymentsRegistrationTestCase(OFVCRMixin, TestCase):
+    VCR_TEST_FILES = TESTS_DIR / "data"
+
+    def setUp(self):
+        super().setUp()
```

Comment:
that's a lot of set up shared between the tests - it looks like you should be able to put it in `setUpTestData` instead of running it for every test?